iFixit/react-commerce
# Update AMEX logo seed image

fixes #229 
 
## Changelog

- Replace AMEX logo seed image

## QA

1. Visit the Vercel PR preview (replace vercel.app with cominor.com) 
2. Visit the parts product list
3. Verify that the AMEX logo shown is the [full version of the blue box logo](https://github.com/iFixit/react-commerce/issues/229#issuecomment-1076505437) (although adapted B&W)
